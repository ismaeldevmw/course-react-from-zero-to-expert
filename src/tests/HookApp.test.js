import React from 'react';
import { shallow } from 'enzyme';
import { HooksApp } from '../HooksApp';

describe('Tests on <HooksApp />', () => {
    
    test('should display correctly', () => {
        const wrapper = shallow( <HooksApp /> );
        expect( wrapper ).toMatchSnapshot();
    })

});