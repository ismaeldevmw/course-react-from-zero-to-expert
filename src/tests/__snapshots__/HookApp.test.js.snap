// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on <HooksApp /> should display correctly 1`] = `
<div>
  <h1>
    Hello world!
  </h1>
</div>
`;
